src/dbsync/api.rs

```diff
@@ -1992,4 +1990,26 @@ mod tests {
         assert_eq!(func_value.mint_slot, real_value.mint_slot);
         assert_eq!(func_value.txhash, real_value.txhash);
     }
-}
+
+    #[tokio::test]
+    #[allow(non_snake_case)]
+    async fn get_utxo_tokens_CMW_81() {
+        let dp = crate::DataProvider::new(crate::DBSyncProvider::new(crate::Config {
+            db_path: dotenv::var("DBSYNC_DB_URL").unwrap(),
+        }));
+
+        let utxo_tokens = super::get_utxo_tokens(
+            dp.provider(), 
+            3312750, 
+            0
+        ).unwrap();
+        assert_eq!(utxo_tokens.len(), 1);
```

Comment:
this is not a good test, it is not checked if the returned tokens are the correct ones and contains the expected information, it is just checked how many tokens are contained no matter which.